Code/Engine/RendererCore/Debug/DebugRenderer.h

```diff
@@ -70,37 +71,59 @@ class EZ_RENDERERCORE_DLL ezDebugRenderer
 
+  /// \brief Renders the corners of a wireframe box for one frame.
   static void DrawLineBoxCorners(const ezDebugRendererContext& context, const ezBoundingBox& box, float fCornerFraction, const ezColor& color, const ezTransform& transform = ezTransform::IdentityTransform());
 
+  /// \brief Renders a wireframe sphere for one frame.
   static void DrawLineSphere(const ezDebugRendererContext& context, const ezBoundingSphere& sphere, const ezColor& color, const ezTransform& transform = ezTransform::IdentityTransform());
 
+  /// \brief Renders an upright wireframe capsule for one frame.
   static void DrawLineCapsuleZ(const ezDebugRendererContext& context, float fLength, float fRadius, const ezColor& color, const ezTransform& transform = ezTransform::IdentityTransform());
 
+  /// \brief Renders a wireframe frustum for one frame.
   static void DrawLineFrustum(const ezDebugRendererContext& context, const ezFrustum& frustum, const ezColor& color, bool bDrawPlaneNormals = false);
 
+  /// \brief Renders a solid box for one frame.
   static void DrawSolidBox(const ezDebugRendererContext& context, const ezBoundingBox& box, const ezColor& color, const ezTransform& transform = ezTransform::IdentityTransform());
 
+  /// \brief Renders the set of filled triangles for one frame.
   static void DrawSolidTriangles(const ezDebugRendererContext& context, ezArrayPtr<Triangle> triangles, const ezColor& color);
 
+  /// \brief Renders the set of textured triangles for one frame.
   static void DrawTexturedTriangles(const ezDebugRendererContext& context, ezArrayPtr<TexturedTriangle> triangles, const ezColor& color, const ezTexture2DResourceHandle& hTexture);
 
+  /// \brief Renders a filled 2D rectangle in screenspace for one frame.
   static void Draw2DRectangle(const ezDebugRendererContext& context, const ezRectFloat& rectInPixel, float fDepth, const ezColor& color);
 
+  /// \brief Renders a textured 2D rectangle in screenspace for one frame.
   static void Draw2DRectangle(const ezDebugRendererContext& context, const ezRectFloat& rectInPixel, float fDepth, const ezColor& color, const ezTexture2DResourceHandle& hTexture);
 
-  static void Draw2DText(
-    const ezDebugRendererContext& context, const ezStringView& text, const ezVec2I32& positionInPixel, const ezColor& color, ezUInt32 uiSizeInPixel = 16, HorizontalAlignment::Enum horizontalAlignment = HorizontalAlignment::Left, VerticalAlignment::Enum verticalAlignment = VerticalAlignment::Top);
+  /// \brief Displays a string in screenspace for one frame.
+  static void Draw2DText(const ezDebugRendererContext& context, const ezFormatString& text, const ezVec2I32& positionInPixel, const ezColor& color, ezUInt32 uiSizeInPixel = 16, HorizontalAlignment::Enum horizontalAlignment = HorizontalAlignment::Left, VerticalAlignment::Enum verticalAlignment = VerticalAlignment::Top);
```

Comment:
What's the benefit of formatstring here?

Reply:
We have many places where we use a stringbuilder first to format something and then pass it to Draw2DText. This way, you can now do this directly with ezFmt("bla {}", 42). It's just more convenient.